source/solvers/navier_stokes_vof_assemblers.cc

```diff
@@ -94,11 +95,12 @@ GLSNavierStokesVOFAssemblerCore<dim>::assemble_matrix(
       const double tau =
         this->simulation_control->get_assembly_method() ==
             Parameters::SimulationControl::TimeSteppingMethod::steady ?
-          1. / std::sqrt(std::pow(2. * density_eq * u_mag / h, 2) +
-                         9 * std::pow(4 * dynamic_viscosity_eq / (h * h), 2)) :
-          1. / std::sqrt(std::pow(sdt, 2) +
-                         std::pow(2. * density_eq * u_mag / h, 2) +
-                         9 * std::pow(4 * dynamic_viscosity_eq / (h * h), 2));
+          calculate_navier_stokes_gls_tau_steady(u_mag,
+                                                 viscosity_eq,
+                                                 h,
+                                                 density_eq) :
```

Comment:
good catch, I like that you are calling a common function! This is so much better than code copying

Also it makes really clearer the `steady?` condition (if true, we use `calculate_navier_stokes_gls_tau_steady`, if false, we use `calculate_navier_stokes_gls_tau_transient`)